samples/samples-springmvc/pom.xml

```diff
@@ -9,8 +9,9 @@
     <version>0.0.1-SNAPSHOT</version>
     <packaging>war</packaging>
     <properties>
-        <spring.version>4.0.5.RELEASE</spring.version>
+        <spring.version>4.3.17.RELEASE</spring.version>
         <jackson.version>2.2.3</jackson.version>
+        <jackson.databind.version>2.8.11.1</jackson.databind.version>
```

Comment:
can we update jackson.version as well?  Just noticed there is no `2.8.11.1` but there is a `2.8.11`

Reply:
Better yet, is there any reason we can't go to 2.9.7?  Seems like both databind and core have that version.

Reply:
Okay, I'll go ahead with 2.9.7 for both databind and core